### DISCUSSION TITLE
Flax not winning during Chia pooling

### DISCUSSION THREAD

**Discussion by BasvanH** [guydavis/machinaris#282] [Q&A] [answered]
Hello,

I noticed that my Flax aint winning anymore since I started Chia pooling. I checked my farmers and the `xfx_target_address` is set to my Flax wallet address.

Any idea what could be going wrong here?

Regards,
Bastiaan

**Comment by guydavis** [comment, marked as answer]
Hmm, did you try tracking the payout after a week using https://flaxexplorer.org/nft?

I recommend you open an issue with [their Github](https://github.com/Flax-Network/flax-blockchain/issues).  I've been finding blocks, but am solo-plotting original plots mostly.  

They say an [update ](https://github.com/Flax-Network/flax-blockchain/releases/tag/0.1.1) is coming soon.

**Reply by guydavis** [reply]
Marking this as the Answer as no further comments received.
